src/app/app/dashboard/page.tsx

```diff
@@ -1,4 +1,34 @@
+import Accordion from "@/components/Accordion";
+
 export default function Page() {
+  //Need to make the data an separated file.
+  const items = [
+    {
+      id: "pd1",
+      label: "Checklist 1",
+      content: "Yes that the first one",
+    },
```

Comment:
We should define a type for these items and share them. This array doesn't now that all of its items have the same format.

```ts
    type TaskItem = {
      id: string
      label: string
      content: string
    },
```